src/components/common/FileUploadPdf.vue

```diff
@@ -205,15 +205,15 @@ export default class FileUploadPdf extends Vue {
     try {
       const arrayBuffer = await file.arrayBuffer()
       const data = new Uint8Array(arrayBuffer) // put it in a Uint8Array
-      const document = await this.pdfjsLib.getDocument({ data: data })
+      const document = await this.pdfjsLib.getDocument({ data }).promise
       const perms = await document.getPermissions()
       return { isEncrypted: false, isContentLocked: !!perms }
-    } catch (err) {
-      if (err.name === 'PasswordException') {
+    } catch (error) {
+      if (error.name === 'PasswordException') {
         return { isEncrypted: true, isContentLocked: true }
       }
-      throw err // re-throw any other error
     }
+    return { isEncrypted: false, isContentLocked: false }
```

Comment:
The only time this statement is executed is after the catch block. I think this is not a safe return value, Thoughts?

Reply:
See comment above.